### DISCUSSION TITLE
struct.error: argument out of range

### DISCUSSION THREAD

**Discussion by DayzenDEV** [flet-dev/flet#3477] [Q&A]
### Question

I want to package my application with Flet, but in the assets folder I have to include some directories that weigh approximately 10.7 Gb, and it seems that the packager does not support it. Is there a way to include large files?

![Captura de pantalla 2024-06-16 114932](https://github.com/flet-dev/flet/assets/172546222/e1183e22-4559-4f3c-ae4e-c2ce3012ed17)

To have more context of my application, it is a search engine which has two directories to choose where to search, and in each directory there are certain videos, which if I do not include them in the packaging it does not find them, if I only include a couple, the packaging works but if I include them all, it throws an error...

![Captura de pantalla 2024-06-16 114837](https://github.com/flet-dev/flet/assets/172546222/e16b1037-003a-4f8a-abe2-02e260be8c5f)

In the code example, I share how I access directories in the code.

### Code sample

```python
# Determine the base directory
BASE_DIR = Path(__file__).resolve().parent

# Define las rutas de los directorios y el ícono personalizado
DIRECTORIO_NUEVO_HIMNARIO = BASE_DIR / 'assets/Nuevo_Himnario'
DIRECTORIO_ANTIGUO_HIMNARIO = BASE_DIR / 'assets/Antiguo_Himnario'
DIRECTORIO_MINIATURAS = BASE_DIR / 'assets/miniaturas'
ICONO_PERSONALIZADO = BASE_DIR / 'assets/Iconos/darktheme.png'

# Define the directories for the radio button options
DIRECTORIES = {
    "Nuevo Himnario": [DIRECTORIO_NUEVO_HIMNARIO],
    "Antiguo Himnario": [DIRECTORIO_ANTIGUO_HIMNARIO],
    "Ambos": [
        DIRECTORIO_NUEVO_HIMNARIO,
        DIRECTORIO_ANTIGUO_HIMNARIO
    ]
}
```

### Error message

```bash
struct.error: argument out of range
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Please share the full logs + full error message here (use --v flag).
